### PR TITLE
Use schema 0.4.0

### PR DESCRIPTION
**Pull request by dignati**
This could solve some of the problems we have, as described in #66

### CODE REVIEW

**Comment by sritchie** [comment]
I think an upgrade to 0.3.x would make more sense; the API changes from 0.4.x are present as well, but we wouldn't be forcing users into a big bump. My big production app isn't on 0.4 yet. Can you try these changes with 0.3? Thanks for this!



—

**Comment by dignati** [comment]
These changes are not neccessary  with schema < 0.4.0. [Changelog](https://github.com/Prismatic/schema/blob/master/CHANGELOG.md#040)
om-tools 0.3.11 requires schema 0.4.0. [Changlelog](https://github.com/Prismatic/om-tools/blob/master/CHANGELOG.md#0311)

So I guess the next version bump would have to be a 0.x.x one.

File: src/om_bootstrap/button.cljs [hunk 1]
```diff
@@ -8,7 +8,7 @@
             [om-tools.dom :as d :include-macros true]
             [om-tools.mixin :refer-macros [defmixin]]
             [schema.core :as s])
-  (:require-macros [schema.macros :as sm]))
+  (:require-macros [schema.core :as sm]))
```
Comment (sritchie) [inline src/om_bootstrap/button.cljs]: We already include `schema.core :as s`. Can you just remove the `require-macros` line and change `sm` to `s`?

**Comment by sritchie** [comment]
Okay, I'm down to make this change. We'll have to change all of the examples on the doc site too.

File: src/om_bootstrap/button.cljs (hunk 1 above)
Comment (sritchie) [inline src/om_bootstrap/button.cljs]: oh, also you'll need to change the `schema.core :as s` line to

```clojure
[schema.core :as s :include-macros true]
```

**Comment by sritchie** [comment]
Thanks for doing this!

**Comment by dignati** [comment]
Okay, now on to the docs.

**Comment by dignati** [comment]
@sritchie Could you point out what needs to be changed in the docs? I can't find any references to schema.

**Comment by sritchie** [comment]
Looks like I was wrong about the docs.

This is great! Thanks a lot. I'll get a new release out now.

**Comment by sritchie** [comment]
Okay, just released 0.5.0. Give it a try and let me know if it fixes your other issues.

**Comment by dignati** [comment]
@sritchie Great, thank you for merging! In fact that fixed all of our issues.

**Pull request merged**